Code/EditorPlugins/VisualScript/EditorPluginVisualScript/VisualScriptClassAsset/VisualScriptClassAsset.cpp

```diff
@@ -77,7 +80,7 @@ ezTransformStatus ezVisualScriptClassAssetDocument::InternalTransformAsset(ezStr
   }
 
   ezStringBuilder sDumpPath;
-  if (false)
+  if (GetProperties()->m_bDumpAST)
```

Comment:
This looks hacky

Reply:
I know, haven't found a better solution yet. Can be cleaned up at some point.